drake/doc/code_style_guide.rst

```diff
@@ -47,12 +47,22 @@ There are several tools for helping you to compile with the required style. They
 ClangFormat
 ^^^^^^^^^^^
 
-(*Notes comming soon*)
+The following instructions were tested on OS X but should also work on Linux. A different installation procedure may need to be followed on Windows.
```

Comment:
I decided to go with `brew` since that's what was recommended here: http://stackoverflow.com/questions/21151731/where-are-clang-format-and-clang-format-py-in-mac-os-x-with-xcode-command-line-t